src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Content/OrganizationListPage.xaml

```diff
@@ -7,174 +7,187 @@
 	  xmlns:toolkit="using:Uno.UI.Toolkit"
 	  mc:Ignorable="">
 
-	<Grid Background="{StaticResource Color07Brush}"
+	<Grid Background="{StaticResource Color02Brush}"
 		  toolkit:VisibleBoundsPadding.PaddingMask="Left,Right">
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition Height="*" />
+		</Grid.RowDefinitions>
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition x:Name="MainGridViewColumn_1"
+							  Width="*" />
+			<ColumnDefinition x:Name="MainGridViewColumn_2"
+							  Width="0" />
+		</Grid.ColumnDefinitions>
 		<VisualStateManager.VisualStateGroups>
 			<VisualStateGroup>
 				<VisualState x:Name="wide">
 					<VisualState.StateTriggers>
 						<AdaptiveTrigger MinWindowWidth="1024" />
 					</VisualState.StateTriggers>
-
 					<VisualState.Setters>
 						<Setter Target="RootGrid.BorderThickness"
 								Value="1,0" />
 						<Setter Target="ReloadButton.MaxWidth"
 								Value="300" />
 						<Setter Target="LargeViewNavigation.Visibility"
 								Value="Visible" />
+						<Setter Target="LargeViewNavigation.(Grid.Row)"
+								Value="0" />
+						<Setter Target="LargeViewNavigation.(Grid.RowSpan)"
+								Value="2" />
 						<Setter Target="HamburgerButton.Visibility"
 								Value="Collapsed" />
-						<Setter Target="ContentView.Grid.Column"
+						<Setter Target="ContentView.(Grid.Column)"
 								Value="1" />
+						<Setter Target="MainGridViewColumn_1.Width"
+								Value="Auto" />
+						<Setter Target="MainGridViewColumn_2.Width"
+								Value="*" />
 					</VisualState.Setters>
 				</VisualState>
 			</VisualStateGroup>
 		</VisualStateManager.VisualStateGroups>
+		<!-- CommandBar -->
+		<CommandBar x:Name="CommandBarView"
+					Content="My Organizations"
+					Grid.Column="0"
+					Grid.ColumnSpan="2">
+			<CommandBar.PrimaryCommands>
+				<!-- Hamburger Button  -->
+				<AppBarButton x:Name="HamburgerButton"
+							  Click="HamburgerButton_Click"
+							  Style="{StaticResource BurgerMenuButtonStyle}">
+				</AppBarButton>
+			</CommandBar.PrimaryCommands>
+		</CommandBar>
+
+		<Grid Grid.Row="1"
+			  x:Name="ContentView">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="*" />
+			</Grid.ColumnDefinitions>
+			<ScrollViewer>
+				<Grid x:Name="RootGrid"
+					  Background="{StaticResource Color02Brush}"
+					  BorderBrush="{StaticResource Color06Brush}">
+			
+					<!-- Loading State -->
+					<Grid Visibility="{Binding Organizations.IsExecuting, Converter={StaticResource TrueToVisible}, FallbackValue=Visible, TargetNullValue=Visible}">
```

Comment:
Are you sure you want `TargetNullValue` ?

Reply:
I guess yes this is something that has been implemented a long time ago... I'll investigate it.